Stellar-contract.x

```diff
@@ -275,5 +335,11 @@ case SCO_CONTRACT_CODE:
     SCContractCode contractCode;
 case SCO_ACCOUNT_ID:
     AccountID accountID;
+case SCO_ACCOUNT:
+    SCAccount account;
+case SCO_ADDRESS:
+    SCAddress address;
+case SCO_NONCE_KEY:
+    SCAddress nonceAddress;
```

Comment:
Why does an address when used as a nonce key need its own type? I would think ScAddress could be used for this without needing to introduce a new object type.

Reply:
This iteration has nonces managed by host and hence we need a special object type to both make it simple for host to work with it and also to not allow contracts to interact with it (similar to `ScContractCode` objects).

Reply:
Since this is a recurring and rather generic problem, maybe we need an ScObject type `Internal` that contains an ScVal?